### DISCUSSION TITLE
「专栏文字选择」功能 B 站已经自带，该组件可删

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#4065] [综合讨论 / General]
如题

**Comment by Geeyun-JY3** [comment]
虽然不清楚这功能一开始是否只用于解除专栏的复制限制（我个人一直在用针对该需求的通用脚本），但这功能现在还是有的用处的，它能去除复制结果中附在最后的 `作者：**** <URL> 出处：bilibili`。
可能只是需要改改名字和描述。

**Reply by Geeyun-JY3** [reply]
猛地发现好像通用的网页限制解除类脚本都能做到这事
